client/app/bundles/course/video/components/tables/VideoTable.tsx

```diff
@@ -40,7 +40,22 @@ const VideoTable: FC<Props> = (props) => {
     return <Note message={intl.formatMessage(translations.noVideo)} />;
   }
 
-  videos.sort((a, b) => a.title.localeCompare(b.title));
+  const videoSortMethodByDateThenTitle = (val1, val2) => {
```

Comment:
Specify the type of `val1` and `val2` in the arguments instead of force-typing those below

Reply:
`val1` and `val2` are also typed wrongly. Isn't it supposed to be `VideoListData`?

Reply:
fixed